src/ui/public/tags/components/tags_select.js

```diff
@@ -152,6 +150,9 @@ export class TagsSelect extends Component {
             value={this.state.search}
             onChange={this.fetchTags}
           />
+          <EuiFormHelpText style={{ paddingBottom: 0 }}>
+            Manage and add tags over in <a href="/app/management/kibana/tags">tag management</a>
```

Comment:
I have no idea how we're doing routing in Kibana React. Can you link this off to the proper place?